src/api.php

```diff
@@ -322,30 +333,63 @@
+                                || preg_match('/^(g)\.([0-9]+)(_([0-9]+))?$/', substr($_GET['search_position'], strpos($_GET['search_position'], ':')), $aRegs))
+                                // $aRegs:        1    2       3 4
+                        ) {
                             if ($aRegs[1] == 'g') {
+                                // The user is looking for a specific genomic position.
+                                if (strpos($_GET['search_position'], ':') !== false) {
+                                    // If a ':' was found, the user specified a genome build.
+                                    list($sGB, $searchPosition) = explode(':', $_GET['search_position']);
+
+                                    if (!isset($aActiveGBs[$sGB])) {
+                                        // The genome build specified by the user is not active in the
+                                        //  database. We will send an error.
+                                        header('HTTP/1.0 404 Not Found');
+                                        die('The given genome build was not found in the database.');
+                                    }
+
+                                } else {
+                                    // The user did not specify a specific genome build.
+                                    if (count($aActiveGBs) == 1) {
+                                        // If there is only one build, we can simply use that one.
+                                        $sGB = array_keys($aActiveGBs)[0]; // Fixme; Can this be done? Function()[]?
+                                    } else {
+                                        // There are multiple active builds, yet the user did not
+                                        //  specify which one to use. We will send an error.
+                                        header('HTTP/1.0 400 Bad Request');
+                                        die('To search genomic positions, please identify which reference genome you want to use.');
+                                    }
+                                }
+
                                 if (empty($aRegs[3])) {
                                     // No range. Absolute location.
                                     $aRegs[4] = $aRegs[2];
                                 }
                                 // Very important in genomic positions: genes on antisense will have positions like g.5678_1234 in the database!!!
-                                $nMin = min($aRegs[2], $aRegs[4]);
-                                $nMax = max($aRegs[2], $aRegs[4]);
+                                $nMin = (int) min($aRegs[2], $aRegs[4]);
+                                $nMax = (int) max($aRegs[2], $aRegs[4]);
+                                $sUnderscorePlusSuffix = (!$aActiveGBs[$sGB]? '' : '_' . $aActiveGBs[$sGB]);
```

Comment:
Hm, is it true you mean to build `$aActiveGBs` with a query like this one:
`SELECT id, if(column_suffix = "", "", CONCAT("/", column_suffix)) as suffix FROM lovd_genome_builds`?

I wanted to do this myself as well, but the problem I found was that the suffix cannot always be concatenated to the same symbol; so it is `/` for the DNA field, but `_` for the position fields. However, I could imagine that maybe something like:
`SELECT id, if(column_suffix = "", "", CONCAT("/", column_suffix)) as slashSuffix, if(column_suffix = "", "", CONCAT("_", column_suffix)) as underscoreSuffix FROM lovd_genome_builds;` to be an improvement!

Maybe I am going in the wrong direction, but this second option seems good to me!
Let me know what you think :)

Reply:
Oops, I had missed this example: 
> E.g., instead of `vog.position_g_start' . $sUnderscorePlusSuffix` we can do `$aActiveGBs[$sGB]['position_start']`.

This is fine as well; I will make it work!

Reply:
Suffix comments are all processed to indeed work with `$aActiveGBs[$sGB][<field>]`.

Reply:
Excellent, thanks!